addons/wp/stringtable.xml

```diff
@@ -255,6 +286,7 @@
             <Polish>Promień</Polish>
             <Russian>Радиус</Russian>
             <Italian>Raggio</Italian>
+            <Chinesesimp>建筑和静态武器半径范围内驻守并配备/n两倍于该数字的人员用于巡逻</Chinesesimp>
```

Comment:
I only google translated this to check as this looked rather long. It seems this is the tooltip of "TaskCamp"_Radius and not the display name.

The following entries are also not correct as they represent the next entry in the string table.